test/unit/controllers/actionbuilders/HeaderValidatorSpec.scala

```diff
@@ -31,6 +31,7 @@ import uk.gov.hmrc.play.test.UnitSpec
 import util.RequestHeaders.{ValidHeadersV2, _}
 import util.{ApiSubscriptionFieldsTestData, TestData}
 
+//TODO MC add v3
```

Comment:
Looks like this is missing V3 tests

Reply:
please note `//TODO: Confirm with QA's that schema tests like these that give confidence around schema changes are tested manually by QA's`
I would leave it as is.